src/docs/forms/components/CheckBox/CheckBoxDescription.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default class CalendarInputDescription extends React.Component {
+    render() {
+        return (
+            <p>
+                CheckBoxes are used to let users select one or more options of a limited number of choices.
```

Comment:
R: "The CheckBox component can be used to let the user select any number of options from a list of preset options."